lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -60,6 +69,48 @@ const char *location_service_get_certificate_nrf_cloud(void)
 	return tls_certificate;
 }
 
+#if defined(CONFIG_NRF_CLOUD_MQTT)
+static void location_service_location_ready_cb(const struct nrf_cloud_cell_pos_result *const result)
+{
+	if (result != NULL) {
```

Comment:
Is it possible that getting the location fails? I guess that's always a possibility. What happens in that case, is the callback called? Is result `NULL`? Are the values just undefined/zero/whatever? This doesn't seem to be documented in `nrf_cloud_cell_pos.h`.